content/docs/gke/deploy/deploy-cli.md

```diff
@@ -74,6 +74,8 @@ Follow these steps to deploy Kubeflow:
     export ZONE=<your target zone> #where the deployment will be created
 
     export PROJECT=<your GCP project ID>
+    
+    # The value of KFAPP must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character. (for example,  'kubeflow-test',  or 'kfw-test').
```

Comment:
Thanks for doing this update! Here's the preview of your change:
https://deploy-preview-934--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-cli/

Please would you break up the comment into two or even three lines, to alleviate the need for scrolling left-right.

Nit: change this:

```
(for example,  'kubeflow-test',  or 'kfw-test').
```

To this (capital "F", removed a space, removed a comma, and moved punctuation inside brackets):

```
(For example,  'kubeflow-test' or 'kfw-test'.)
```